cloud/azure/sfx/integrations-azure.tf

```diff
@@ -17,3 +17,24 @@ resource "signalfx_azure_integration" "azure_integration" {
   tenant_id     = var.azure_tenant_id
   subscriptions = var.azure_subscription_ids
 }
+
+# "Named token to use for ingest on the SignalFx Azure integration"
+resource "signalfx_org_token" "azure_integration" {
+  name        = local.integration_name
+  description = "Org token for ingesting data from ${local.integration_name} Azure integration"
+
+  # Where to send notifications about this token's limits. Please consult the Notification Format laid out in detectors
+  # notifications = ["Email,foo-alerts@bar.com"]
+
+  # host_or_usage_limits {
```

Comment:
Can it be a variable to allow to override default settings ?

Reply:
yes it is planned !

but I would like to merge this one and do that in another PR for all cloud integrations (including gcp and aws)